spec/string-functions/quote.hrx

```diff
@@ -0,0 +1,80 @@
+<===> unquoted/input.scss
+a {
+  a: quote(a);
```

Comment:
Style nits:

* Placeholder names [should be alphabetical](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#do-use-single-letter-names-for-irrelevant-placeholders), rather than all being `a`.

* Rules these small should be [one-liners](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#prefer-one-line-rules-when-they-contain-a-single-child-with-no-children).

For example:

```scss
a {b: quote(c)}
```

Reply:
Done